.github/workflows/check.yml

```
@@ -50,18 +50,8 @@ jobs:
           rustup update stable
           rustup component add llvm-tools-preview
           cargo install grcov
+          cargo install rust-covfix
 
       - name: Test coverage
         run: |
-          export CARGO_INCREMENTAL=0
-          export SKIP_BUILD_WASM=true
-          export BUILD_DUMMY_WASM_BINARY=true
-          export LLVM_PROFILE_FILE="llvmcoveragedata-%p-%m.profraw"
-          export WASM_TARGET_DIRECTORY=/tmp/wasm
-          cargo build --features all-nodes
-          export RUSTFLAGS="-Zinstrument-coverage"
-          rm -rf target/debug
-          cargo test --all --features all-nodes
-          grcov . --binary-path ./target/debug/ -s . -t lcov --branch --ignore-not-existing --ignore "/*" -o lcov.info
-          bash <(curl -s https://codecov.io/bash) -f lcov.info
-          
\ No newline at end of file
+          sh ./maintain/test-coverage/test-coverage.sh
```

Caller is here.  Probably intended

```suggestion
          sh ./.maintain/test-coverage/test-coverage.sh
```

---

Sorry, My mistake. Fixed.